src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadRespondentHandlerTest.java

```diff
@@ -1190,10 +1192,10 @@ void shouldBreakWhenThereIsAnyCaseBundlesWithNullCreatedDate() {
         CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
 
         // When: handler is called
-        // Then: an exception is thrown
-        assertThrows(NullPointerException.class, () -> {
-            handler.handle(params);
-        });
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+        CaseData updatedData = mapper.convertValue(response.getData(), CaseData.class);
+        // Then: respondent docs uploaded after bundle should return size 0
+        assertThat(updatedData.getRespondentDocsUploadedAfterBundle()).isNull();
```

Comment:
If you are changing the test, it means you are changing the logic.
We are not supposed to change the logic at this stage.

Please revert this test change and the changes that required this modification to the test.